format/FormatNumber.js

```diff
@@ -316,6 +316,17 @@ function isInvalidInput(v) {
     return v == null || v === '';
 }
 
+function processTipFn(tipFn, originalValue) {
+    if (tipFn === true) {
+        const formatter = numberRenderer(originalValue, {ledger: true, forceLedgerAlign: false, precision: 12, zeroPad: false});
+        return formatter(originalValue);
```

Comment:
We don't want to create a renderer here just to call it - we can simply call `fmtNumber(v, {...theFmtOpts}`.

The renderer functions (e.g. `numberRenderer`) have a different signature: they _only_ take the options object and return a curried function that then _only_ takes the value to be formatted. 

Given that, this code isn't really working as you think it is - the renderer that's returned and assigned to `formatter` is a version of `fmtNumber` with all of its default options. The options you pass in as a second argument on line 321 aren't being respected - they are just silently dropped in a very javascript-y way.

Try configuring `tipFn:true` on a grid column that shows negative numbers - e.g. [the P&L col in Toolbox's SampleGrid](https://github.com/exhi/toolbox/blob/b27a1c976fa52af4bf0b8e0ed4073c628c265204/client-app/src/desktop/common/SampleGrid.js#L78) and you will see that the tip values are not formatted in ledger format.

Also:
* We have a constant in the FormatNumber class for `MAX_NUMERIC_PRECISION` we can use instead of the 12 (sorry I know that number was directly from the ticket).
* If we pass this function the original opts, we can use the ledger formatting only if it was requested for the overall formatter.  I think that makes sense - if a ledger format is not desired / expected for the actual output, it would be strange to appear in the tip.
  * I think however that ledger is the only opt that makes sense to consult at this time in the tip use-case...

Reply:
Done! Code updated, committed and pushed. 

Thanks for the feedback. Much appreciated.